be/src/common/config.h

```diff
@@ -645,6 +645,10 @@ CONF_Int64(pipeline_yield_max_time_spent, "100000000");
 CONF_Int64(pipeline_scan_thread_pool_thread_num, "0");
 // queue size of scan thread pool for pipeline engine.
 CONF_Int64(pipeline_scan_thread_pool_queue_size, "102400");
```

Comment:
when merge  to main branch，would better  remove these configs

Reply:
@kangkaisen  `io_dispatcher_thread_pool` also uses `pipeline_scan_thread_pool_thread_num`. Let me remove `pipeline_scan_thread_pool_queue_size`.